tests/realtikvtest/importintotest/import_into_test.go

```diff
@@ -1166,3 +1166,20 @@ func (s *mockGCSSuite) TestAnalyze() {
 	}, 60*time.Second, time.Second)
 	s.tk.MustQuery("SHOW ANALYZE STATUS;").CheckContain("analyze_table")
 }
+
+func (s *mockGCSSuite) TestZeroDateTime() {
+	s.tk.MustExec("DROP DATABASE IF EXISTS import_into;")
+	s.tk.MustExec("CREATE DATABASE import_into;")
+
+	s.tk.MustExec("create table import_into.zero_time_table(t datetime)")
+	s.tk.MustExec("set global sql_mode='STRICT_TRANS_TABLES'")
```

Comment:
```suggestion
	s.tk.MustExec("set @@sql_mode='STRICT_TRANS_TABLES'")
```

more reasonable to set it for this session

Reply:
need reset it after this test, as `tk` is shared